src/components/MovieDetails.js

```diff
@@ -11,7 +11,7 @@ class MovieDetails extends Component {
       id: data.movie.id,
       user: data.user,
       movie: {},
-      isFavorite: data.user && data.user.favorites.includes(data.movie) ? true : false,
+      isFavorite: data.user.favorites.some(favorite => favorite.id === data.movie.id) ? true : false,
```

Comment:
AWESOME use of .some here!! Never really get to use that so I think this is great!